Source/WebKit/Shared/WTFArgumentCoders.serialization.in

```diff
@@ -20,7 +20,7 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-class WTF::URL {
+[WebKitPlatform] class WTF::URL {
```

Comment:
Out of curiosity, what does this do? First time I see it.

Reply:
It puts the symbol in the library libWebKitPlatform.a, which is statically linkable by DRT/WKTR/TWKA for testing.